File: server/src/model/BulletCard.java
package model;

import com.smartfoxserver.v2.protocol.serialization.SerializableSFSType;

public class BulletCard extends Card implements SerializableSFSType {
	/**
	 * if fired == false then the bullet has not been fired ie. it is in the revolver of the bandit it belongs to
	 * else if fired == true then the bullet has been fired ie. it is in the card pile/hand of the bandit it belongs to
	 */
	public boolean fired;
	public String belongsToAsString;
		
	//--EMPTY CONSTRUCTOR FOR SERIALIZATION--
	public BulletCard() {}
	
	public BulletCard(String belongsTo) {
		this.belongsToAsString = belongsTo;
		this.fired = false;
	}
	
	//fired
	public boolean getFired() {
		return this.fired;
	}
	public void setFired(boolean b) {
		this.fired = b;
	}
	
	//belongsToAsString
    public String getBelongsToAsString() {
        return this.belongsToAsString;
    }

    public void setBelongsToAsString(String belongsTo) {
        this.belongsToAsString = belongsTo;
    }
}

File: colt-express-jan-2021/Assets/Scripts/model/BulletCard.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using Sfs2X;
using Sfs2X.Logging;
using Sfs2X.Util;
using Sfs2X.Core;
using Sfs2X.Entities;
using Sfs2X.Entities.Data;
using Sfs2X.Protocol.Serialization;


//The following code is executed right after creating the SmartFox object:
// using System.Reflection;
//        DefaultSFSDataSerializer.RunningAssembly = Assembly.GetExecutingAssembly();
namespace model {
    public class BulletCard : Card, SerializableSFSType {
    
        public bool fired;
        public string owner; // added 
        public int size; // added by annie

        // --EMPTY CONSTRUCTOR FOR SERIALIZATION--
        public BulletCard() {}

        public BulletCard(string belongsToAsString) {
            base.belongsToAsString = belongsToAsString;
            this.fired = false;
        }

        //fired
        public bool getFired(){
            return this.fired;
        }

        public void setFired(bool b){
            this.fired = b;
        }

        // added by annie
        public int getSize(){
            return this.size; 
        }
    }
}
